src/identitas/proquint.clj

```diff
@@ -15,11 +15,11 @@ equivalent."
 (def ^:private int-to-vowel
   '[\a \i \o \u])
 
-(def ^:private mask-4 0xF0000000)
-(def ^:private mask-2 0xC0000000)
+(def ^:private mask-4 (unchecked-int 0xF0000000))
```

Comment:
Caused by java.lang.IllegalArgumentException
   Value out of range for int: 4026531840